ChakraCore Samples/Hello World/Python/README.md

```diff
@@ -0,0 +1,24 @@
+# ChakraCore Hello-world Python Sample
+This sample shows how to embed ChakraCore with [JavaScript Runtime (JSRT) APIs](http://aka.ms/corejsrtref), and complete the following tasks:
+
+1. Initializing the JavaScript runtime and creating an execution context.
+2. Running scripts and handling values.
+3. Printing out script results.
+
+## Requirements
+This sample requires Python 2.7+ is installed.
+
+## Build and run the sample
+1. Follow the instructions to build ChakraCore (https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore)
+2. Update the sample `chakraCore = CDLL("./BuildLinux/Debug/libChakraCore.so")` with the location of `CharakCore.dll` or .so file
```

Comment:
Update the *line* `chakraCore = CDLL("./BuildLinux/Debug/libChakraCore.so")` with the location of `CharakCore.dll` or .so file

Also, add a link to *line*.